fastlane/docs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+repoName="appcues-android-sdk"
+initialBranch=$(git rev-parse --abbrev-ref HEAD)
+# versionFile="./Sources/AppcuesKit/Version.swift"
+
+# # get last line in Version.swift
+# versionLine=$(tail -n 1 $versionFile)
+# # split at the =
+# version=$(cut -d "=" -f2- <<< "$versionLine")
+# # remove quotes and spaces
+# version=$(sed "s/[' \"]//g" <<< "$version")
```

Comment:
planning to take another pass on this with a version release script, coming up.  It will handle packaging up the release to maven, adding the release in github, and generating these docs.  Modeling after what was done on iOS.